modules/ssr-benchmarks/tsconfig.app.json

```
@@ -0,0 +1,10 @@
+/* To learn more about this file see: https://www.typescriptlang.org/docs/handbook/tsconfig-json.html. */
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../out-tsc/app",
+    "types": ["node"]
+  },
+  "files": ["src/main.ts", "src/main.server.ts", "main.ts"],
+  "include": ["src/**/*.d.ts", "../../node_modules/@types/dom-navigation/index.d.ts"]
```

Is `dom-navigation` needed? If so, reference types using the `types` options. 


```suggestion
{
  "extends": "./tsconfig.json",
  "compilerOptions": {
    "outDir": "../../out-tsc/app",
    "types": ["node", "dom-navigation"]
  },
  "files": ["src/main.ts", "src/main.server.ts", "main.ts"],
  "include": ["src/**/*.d.ts"]
```